content/_guides/faq.md

```diff
@@ -11,10 +11,25 @@ seo:
 
 ## How to find Libera Chat staff?
 
-Most [staff](/about#wider-staff-and-organisation-membership) have the
-Operator mode in `#libera` to make them easy to find when they're connected.
-There are also some robots, currently `ChanServ` and `ozone`, with that mode.
-The robots do not answer questions.
+### Immediately available staff
+
+[Staff](/about#wider-staff-and-organisation-membership) make use of a list
+to highlight staff members who are immediately available. This allows you to
+find and contact staff discretely and is our preferred method of taking
```

Comment:
```suggestion
find and contact staff discreetly and is our preferred method of taking
```

Both words make sense, but if you did want `discretely` there then I'd perhaps suggest `individually` instead so other people don't think it's a typo.